tedx-website/src/components/speakerlist.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PageTitle from "../page_title/page_title";
+import speakersData from "../../data/speakers_data"
+import Speaker from "../speaker/speaker"
+
+function SpeakerList(props) {
+    const speakers = speakersData;
+
+    return (
+        <div className="speakers-page page-container">
+            <PageTitle title="Speakers" />
+            {props.map(speakers => (
```

Comment:
I don't think `props.map` should work here. Did it work on your end? `props` isn't the data we want, so we shouldn't be mapping over it. From my understanding, we should be using `speakers.map` here, or even better don't define `speakers` and directly use `speakersData.map`